observability/metrics/src/main/scala/com/daml/metrics/api/noop/Metrics.scala

```diff
@@ -6,8 +6,8 @@ package com.daml.metrics.api.noop
 import java.time.Duration
 import java.util.concurrent.TimeUnit
 
-import com.daml.metrics.api.MetricHandle.Timer
 import com.daml.metrics.api.MetricHandle.Timer.TimerHandle
+import com.daml.metrics.api.MetricHandle.{Counter, Gauge, Histogram, Meter, Timer}
 import com.daml.metrics.api.MetricsContext
 
 sealed case class NoOpTimer(name: String) extends Timer {
```

Comment:
Any reason to have `sealed` on this case class ?

Reply:
Indeed, no real benefit.

Reply:
Will remove it